server/remote/msgraph/subscription.go

```diff
@@ -13,12 +15,19 @@ import (
 
 const subscribeTTL = 10 * time.Minute
 
+func newRandomString() string {
```

Comment:
Just out of curiosity, why not a new guid?

Reply:
A GUID wouldn't be a full 128 characters, and I thought there might be some advantage in security or randomness by not using a GUID. On the other hand, a GUID should technically work.

Reply:
They are 128 bits, often encoded as a 32-character hex value.

Reply:
The `ClientState` can be up to 128 characters so I thought I'd use the full length, although it will accept a shorter string.

Reply:
(base64 will encode 96 bytes into a 128-character string)

Reply:
And for the reasons listed here: https://devblogs.microsoft.com/oldnewthing/20150701-00/?p=45241